pkg/cmd/auth/logout/logout.go

```diff
@@ -39,7 +39,19 @@ func NewCmdLogout(f *cmdutil.Factory, runF func(*LogoutOptions) error) *cobra.Co
 			for an account. The authentication configuration is only
 			removed locally.
 
-			This command does not invalidate authentication tokens.
+			This command does not revoke authentication tokens.
+
+			To revoke all authentication tokens generated by the GitHub CLI:
+
+			1. Visit <https://github.com/settings/applications>
+			2. Select the "GitHub CLI" application
+			3. Select "Revoke Access"
+			4. Select "I understand, revoke access"
+
+			Note: this procedure will revoke all authentication tokens ever
+			generated by the GitHub CLI across all your devices.
+
+			For more information about revoking OAuth application tokens, see <https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps>
```

Comment:
I'll defer to you, @BagToad, whether starting the URL on its own line is easier to read or not:

```suggestion
			For more information about revoking OAuth application tokens, see:
			<https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps>
```